roles/template_service_broker/tasks/deploy.yml

```diff
@@ -48,10 +48,15 @@
     {{ openshift_client_binary }} process --config={{ mktemp.stdout }}/admin.kubeconfig -f "{{ mktemp.stdout }}/{{ __tsb_rbac_file }}"
     | {{ openshift_client_binary }} auth reconcile --config={{ mktemp.stdout }}/admin.kubeconfig -f -
 
+# TSB Route
+- shell: >
```

Comment:
Please use command instead of shell.

Also, please give the command a name, it aides in troubleshooting if the command were to fail.

Other than that, this looks pretty good to me.

Reply:
@michaelgugino Updated